docs/REVIEWING.md

```diff
@@ -0,0 +1,101 @@
+  other ways to contribute, too.
+
+- For the Threadbare maintainers, community reviews of pull requests helps us to
+  focus on areas where our attention and assistance is particularly needed. We
+  are just two developers, and there are only so many hours in the day, so
+  community review is much appreciated!
+
+## Playtesting
+
+Play the game, focusing on the scenes related to the proposed change:
+
+- Does the game build and run correctly?
+
+- If it is a new quest, is the quest playable?
+
+- Has the proposed change introduced bugs elsewhere in the game? (For example, a
+  change to a shared component may fix a bug in one level, but introduce a
+  problem in another.)
+
+In general, a contribution should broadly improve or add to the game.
+It's fine for a contribution to need further work in future; for example, a
+StoryQuest may have only one challenge implemented, or a larger change to the
+game's base resolution may need further testing and follow-up changes. Use your
+judgement.
+
+## Style, legal, and attribution
+
+Pull requests should follow the guidelines laid out in the [contribution
+guidelines](./CONTRIBUTING.md). In brief:
+
+- In-game text and dialogue should be written in US English, with the exception
+  of StoryQuests.
+
+- The coding style checks, licensing checks, and automated web build should
+  succeed.
+
+- The provenance of all third-party assets and code must be identified, and
+  their license checked to ensure we have the legal right to distribute them in
+  the game.
+
+- The pull request title and description must be clearly written, identifying
+  any issues it will fix.
+
+- Any co-authors are identified, following [Creating a commit with multiple
+  authors][co-authored-by], if they did not author any of the commits in the
+  pull request.
+
+[co-authored-by]: https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors
+
+## Fixing issues on behalf of the submitter
+
+Threadbare contributors often have little to no previous experience of
+contributing to an open-source project using Git. We are delighted to receive
+these contributions!
+
+Unlike in some open-source projects, where the submitter is expected to make all
+requested changes themself, the Threadbare maintainers may choose to fix issues
+directly, on behalf of the submitter. We do this to reduce the number of times
+that a pull request must be bounced between submitter and reviewer; and because
+it is sometimes more efficient to make a change than to describe it.
+
+For example, if the pull request is ready to merge except for its title and
+description not matching our style, the maintainer may rewrite the title and
+description, then merge the pull request; if we notice a typo, the maintainer
+may fix it themself, then merge the pull request. We will always explain in
+the pull request why we have made such changes.
```

Comment:
...with the aim of aiding the contributor in providing an even higher quality contribution next time.
(or something like that)